gensim/models/coherencemodel.py

```diff
@@ -164,8 +187,28 @@ def get_coherence(self):
             return self.aggr(confirmed_measures)
 
         elif self.coherence == 'c_v':
+            if self.window_size is None:
```

Comment:
can we have just two ifs - whether it is window-based or not? It seems some code duplication here.